cpp/velox/compute/VeloxBackend.cc

```diff
@@ -120,79 +120,49 @@ void VeloxBackend::printConf(const std::unordered_map<std::string, std::string>&
 
 void VeloxBackend::init(const std::unordered_map<std::string, std::string>& conf) {
   // Init glog and log level.
-  {
-    auto vlogLevel = stoi(getConfigValue(conf, kGlogVerboseLevel, kGlogVerboseLevelDefault));
-    auto severityLogLevel = stoi(getConfigValue(conf, kGlogSeverityLevel, kGlogSeverityLevelDefault));
-    FLAGS_v = vlogLevel;
-    FLAGS_minloglevel = severityLogLevel;
-    FLAGS_logtostderr = true;
-    google::InitGoogleLogging("gluten");
-  }
+  auto veloxmmecfg = std::make_shared<facebook::velox::core::MemConfigMutable>(conf);
```

Comment:
Looks like a typo? s/veloxmmecfg/veloxmemcfg/ ?

Reply:
Thank you, good catch